lightstreamer_adapter/server.py

```diff
@@ -180,7 +180,7 @@ def _do_run(self, sock):
                 tokens = buffer.splitlines(keepends=True)
                 buffer = ''
                 for token in tokens:
-                    if (token.endswith('\r\n')):
+                    if token.endswith('\n'):
```

Comment:
You should use `os.linesep` from the `os` module in stdlib, this way it'll work only on linux.